### PR TITLE
[stable/prometheus-mongodb-exporter]Chart for mongodb exporter

### PR DESCRIPTION
**Pull request by ananthunair**
#### What this PR does / why we need it:

Currently there is no mongodb exporter chart available as standalone. This should be used indepentant of the exporter

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. A few things from quickly looking over it:

* Please follow our review guidelines:
  https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md
* Please compare with what you get using `helm create` and update accordingly
* I'd suggest you move the chart to stable directly.

**Comment by ananthunair** [comment]
Hi @unguiculus,

Can you review and see if the changes look good? 

Thanks in advance

**Comment by juan131** [comment]
Hi @ananthunair 

I'm not the proper person to review this contribution since it's a new chart and only the approvers listed in the OWNERS file below can approve this kind of PRs:

https://github.com/helm/charts/blob/master/OWNERS

**Comment by ananthunair** [comment]
/assign foxish
Hi @foxish ,
Can you take a look at this PR?

File: stable/prometheus-mongodb-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+kind: Deployment
+metadata:
+  name: {{ include "prometheus-mongodb-exporter.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "prometheus-mongodb-exporter.name" . }}
+    helm.sh/chart: {{ include "prometheus-mongodb-exporter.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```
Comment (steven-sheehy) [inline stable/prometheus-mongodb-exporter/templates/deployment.yaml:11]: values.yaml has replicas, not replicaCount

File: stable/prometheus-mongodb-exporter/values.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Default values for prometheus-mongodb-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ananthunair/mongodb_exporter
```
Comment (steven-sheehy) [inline stable/prometheus-mongodb-exporter/values.yaml:8]: This should default to the official percona/mongodb_exporter image as its better maintained and community supported, not one that you maintain. Already, your branch is 19 commits behind.

File: stable/prometheus-mongodb-exporter/values.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+nameOverride: ""
+fullnameOverride: ""
+
+replicas: 1
+
+service:
+  type: ClusterIP
+  port: 9216
+  targetPort: 9216
```
Comment (steven-sheehy) [inline stable/prometheus-mongodb-exporter/values.yaml:20]: Remove targetPort as its not used and shouldn't really be customizable

File: stable/prometheus-mongodb-exporter/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          env:
+            - name: MONGODB_URL
+              value: {{ .Values.config.datasource }}
+              {{- if .Values.config.auth }}
+            - name: HTTP_AUTH
+              value: {{ .Values.config.auth }}
```
Comment (steven-sheehy) [inline stable/prometheus-mongodb-exporter/templates/deployment.yaml:31]: This variable is missing from the values.yaml. Also, give an example of the format as it's unclear.

File: stable/prometheus-mongodb-exporter/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+              {{- end}}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 80
```
Comment (steven-sheehy) [inline stable/prometheus-mongodb-exporter/templates/deployment.yaml:37]: Why port 80 is exposed? It looks like the default listening port for exporter is [9216](https://github.com/percona/mongodb_exporter/blob/61e655e36c5ce4215d1732ce769fb591b82aa7b5/mongodb_exporter.go#L46).

File: stable/prometheus-mongodb-exporter/values.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+annotations:
+  prometheus.io/scrape: "true"
```
Comment (steven-sheehy) [inline stable/prometheus-mongodb-exporter/values.yaml:23]: Should these annotations be on the service instead of the deployment?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
Missing README.md

**Pull request closed without merging**